Feature/new db values

julia-gatina opened the pull request:
- Updates for restaurants.sql to feature 3 different restaurants;
- Added new values to menu-items.sql to have various V/Veg/GF options;
- Updated README file.

Commit:
Add new values to restaurant table, update images for existing value.

Commit:
Add new values to menu-items.sql with, update existing with longer description.

Commit:
Update README file

Emily-Waters reviewed (changes requested):
db/seeds/02_restaurants.sql - Typo - line 3 - 1324 Main Str => 1324 Main St.
db/seeds/02_restaurants.sql - Typo - line 4 - 510 Burrars Str => 510 Burrard St.

db/seeds/03_menu_item.sql - Typo - line 10 - Plant based sashimi => Plant Based Sashimi
db/seeds/03_menu_item.sql - Typo - line 11 - Miso soup => Miso Soup
db/seeds/03_menu_item.sql - Typo - line 12 - Vegetarian Japanese curry => Vegetarian Japanese Curry

A few small typos in important fields. Some minor ones in other fields but I'm not too concerned with those for now, mainly just the ones that will be prominently displayed.

Otherwise, great work!

Commit:
Fix typo and update menu items name and description

julia-gatina commented:
> db/seeds/02_restaurants.sql - Typo - line 3 - 1324 Main Str => 1324 Main St. db/seeds/02_restaurants.sql - Typo - line 4 - 510 Burrars Str => 510 Burrard St.
> 
> db/seeds/03_menu_item.sql - Typo - line 10 - Plant based sashimi => Plant Based Sashimi db/seeds/03_menu_item.sql - Typo - line 11 - Miso soup => Miso Soup db/seeds/03_menu_item.sql - Typo - line 12 - Vegetarian Japanese curry => Vegetarian Japanese Curry
> 
> A few small typos in important fields. Some minor ones in other fields but I'm not too concerned with those for now, mainly just the ones that will be prominently displayed.
> 
> Otherwise, great work!

Thanks, fixed typos and updated menu items name and some dish descriptions.

Emily-Waters reviewed (approved):
Thanks for making the changes!
